src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/FallbackWindowsRazorProjectHostTest.cs

```diff
@@ -387,9 +388,9 @@ public async Task OnProjectChanged_ReadsProperties_InitializesProject()
         Assert.Same(FallbackRazorConfiguration.MVC_2_0, snapshot.Configuration);
 
         Assert.Collection(
-            snapshot.DocumentFilePaths,
-            filePath => Assert.Equal("C:\\Path\\Index.cshtml", filePath),
-            filePath => Assert.Equal("C:\\Path\\About.cshtml", filePath));
+            snapshot.DocumentFilePaths.OrderBy(d => d),
```

Comment:
Interesting that this hasn't broken before. I guess just changing the comparer did it?

Reply:
Yeah, I guess so. Perhaps its stored in hashcode order, and we happened to get lucky with the previous hashcode calculation?